pandas_bokeh/plot.py

```diff
@@ -1742,6 +1823,57 @@ def line(self, x=None, y=None, **kwargs):
         """
         return self(kind="line", x=x, y=y, **kwargs)
 
+    def step(self, x=None, y=None, **kwargs):
+        """
+        Plot DataFrame columns as step lines.
+
+        This function is useful to plot step lines using DataFrame's values
+        as coordinates.
+
+        Parameters
+        ----------
+        x : int or str, optional
+            Columns to use for the horizontal axis.
+            Either the location or the label of the columns to be used.
+            By default, it will use the DataFrame indices.
+        y : int, str, or list of them, optional
+            The values to be plotted.
+            Either the location or the label of the columns to be used.
+            By default, it will use the remaining DataFrame numeric columns.
```

Comment:
maybe also add the **mode** keyword here, because this seems to be very specific and important for a stepplot.